docs/contracts/PR_508_RISK_MITIGATIONS.md

```
@@ -0,0 +1,42 @@
+# PR-508: Risk Mitigations
+
+**Date:** 2026-01-10
+**Status:** Pending verification (do not merge until gates pass)
+
+### Merge gates (must be green)
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)